config/envs/default.py

```diff
@@ -1,45 +1,47 @@
-import logging
 import os
 from pathlib import Path
 
+from fsd_utils import CommonConfig
 from fsd_utils import configclass
 
 
 @configclass
 class DefaultConfig:
-    """
-    Application Config
-    """
 
-    SECRET_KEY = os.environ.get("SECRET_KEY") or "dev"
-    SESSION_COOKIE_NAME = (
-        os.environ.get("SESSION_COOKIE_NAME") or "session_cookie"
-    )
+    # ---------------
+    #  General App Config
+    # ---------------
+
+    SECRET_KEY = CommonConfig.SECRET_KEY
+    SESSION_COOKIE_NAME = CommonConfig.SESSION_COOKIE_NAME
+    FLASK_ROOT = str(Path(__file__).parent.parent.parent)
+    FLASK_ENV = CommonConfig.FLASK_ENV
+
+    FORCE_HTTPS = CommonConfig.FORCE_HTTPS
+    FSD_LOG_LEVEL = CommonConfig.FSD_LOG_LEVEL
```

Comment:
Maybe log level should be a dict based on FLASK_ENV? eg. `CommonConfig.FSD_LOG_LEVEL[FLASK_ENV]` so we have common log levels per env? Just a thought for discussion

Reply:
Done this as per discussion - now retrieves it based on a dictionary in CommonConfig